bundles/countries-rest-api/composer.json

```diff
@@ -0,0 +1,42 @@
+{
+  "name": "fond-of-oryx/countries-rest-api",
+  "description": "",
+  "license": "MIT",
+  "authors": [
+    {
+      "name": "Pascal Fischer",
+      "email": "p-fischer@gmx.net"
+    }
+  ],
+  "require": {
+    "php": ">=8.0",
+    "spryker/checkout-rest-api-extension": "^1.3.0",
+    "spryker/glue-application": "^1.0.0",
+    "spryker/glue-application-extension": "^1.0.0",
+    "spryker/kernel": "^3.30.0"
```

Comment:
`spryker/kernel` and `spryker/glue-application` are probably not necessary here since they come as dev dependency in `glue-application-extension` and we can assume that those are present in applications anyways